pom.xml

```diff
@@ -4,27 +4,28 @@
     <parent>
         <groupId>org.mule.extensions</groupId>
         <artifactId>mule-core-modules-parent</artifactId>
-        <version>1.1.9</version>
+        <version>1.5.0</version>
```

Comment:
Please try it out and let me know if this works. Cheers!

Reply:
I don't think this was the only reason. I mean, increasing to 1.5.0 it may have been, but we still had to increase it to a parent that was adding the dependencies for 4.3.0 due to the fact that the spring dependencies were colliding.

Reply:
Moreover, parent 1.1.9 does not have supportingJavaVersions at all...how can we call it using reflection if it does not exist. TBH, I don't really get your point here.